src/main/resources/plugin.yml

```diff
@@ -31,6 +31,8 @@ permissions:
     default: false
   multispawn.noteleport:
     default: op
+  multispawn.noteleportondeath:
+    default: op
```

Comment:
Maybe `default: false` on this one? I realize it deviates from `.noteleport` but I feel like it should be the default for all players to be functionally the same, rather than OPs behaving differently.